doc/community-build/Generate-Cortx-Build-Stack.md

```diff
@@ -11,6 +10,8 @@ To know about various CORTX components, see [CORTX Components guide](https://git
 
 ## Procedure
 
+#### **For CentOS 7.8**
```

Comment:
I guess the steps for `CentOS 7.8` and `CentOS 7.9` are exactly the same only the image tag is different. Maybe we can only mention using the appropriate tag as per the OS required.